Engine/source/gui/worldEditor/guiConvexShapeEditorCtrl.cpp

```diff
@@ -1075,7 +1075,7 @@ void GuiConvexEditorCtrl::renderScene(const RectI & updateRect)
          Point3F boxPos = objBox.getCenter();
          objMat.mulP( boxPos );
          
-         drawer->drawObjectBox( desc, objBox.getExtents(), boxPos, objMat, ColorI::WHITE );
+         drawer->drawObjectBox( desc, objBox.getExtents() / 2, boxPos, objMat, ColorI::WHITE );
```

Comment:
This doesn't seem right. Need to figure out why the dimensions are twice the size than expected.
I suspect that this couple lines above has something to do with it:
```cpp
objBox.scale( mConvexSEL->getScale() );
```

Reply:
we';ll revisit the /2 when more time allows